File: app/src/main/java/com/karthik/splash/Models/Oauth/UserAuth.java
package com.karthik.splash.Models.Oauth;

import com.google.gson.annotations.SerializedName;

/**
 * Created by karthikrk on 29/11/17.
 */

public class UserAuth {
    @SerializedName("access_token")
    public String accessToken;
    @SerializedName("token_type")
    public String tokenType;
    @SerializedName("scope")
    public String scope;
    @SerializedName("created_at")
    public String createdAt;
}

File: app/src/main/java/com/karthik/splash/models/oauth/UserAuth.kt
package com.karthik.splash.models.oauth

import com.google.gson.annotations.SerializedName

/**
 * Created by karthikrk on 29/11/17.
 */

data class UserAuth(
        @SerializedName("access_token") var accessToken: String? = null,
        @SerializedName("token_type") var tokenType: String? = null,
        @SerializedName("scope") var scope: String? = null,
        @SerializedName("created_at") var createdAt: String? = null
)
